src/engraving/dom/timesig.cpp

```diff
@@ -272,7 +273,15 @@ PropertyValue TimeSig::propertyDefault(Pid id) const
     case Pid::TIMESIG_TYPE:
         return int(TimeSigType::NORMAL);
     case Pid::SCALE:
-        return style().styleV(Sid::timesigScale);
+    {
+        switch (timeSigPlacement()) {
+        case TimeSigPlacement::NORMAL: return style().styleV(Sid::timeSigNormalScale).value<ScaleF>();
```

Comment:
Would it be an idea to implement this using `getPropertyStyle`? (assuming that the idea is that `Pid::SCALE` belongs to a different `Sid` depending on the placement, and that the purpose of `getPropertyStyle` is exactly for implementing such constructions)

Reply:
Good idea, thanks!